CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+## v3.2.0 - 2022/03/02
+
+- Added support for Python 3.10
+- Dropped support for Python 3.6 (EOL)
+- Dropped support for Django 3.1 and 3.2 (EOL)
```

Comment:
3.2?

Reply:
Good catch. This should read "3.0 and 3.1" instead.

Reply:
https://github.com/dfunckt/django-rules/pull/163